components/mm-components/modal/MmModal.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div
+    v-if="value"
+    :id="id || randomId"
+    class="mm-modal-backdrop"
+    @click="close"
+  >
+    <div class="mm-modal" tabindex="-1" role="dialog" @click.stop="">
```

Comment:
Looks like the only thing that this is missing is a label for a11y. See the attached SS and links

<img width="936" alt="Screenshot 2022-11-23 at 14 03 13" src="https://user-images.githubusercontent.com/1762902/203472427-43627a8e-abbc-473c-88d3-d2702cfa2e7d.png">

https://dequeuniversity.com/rules/axe/4.4/aria-dialog-name
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role